python-logging/logging_stack_traces.py

```diff
@@ -0,0 +1,17 @@
+import logging
+
+logging.basicConfig(
+    filename="app.log",
+    encoding="utf-8",
+    filemode="a",
+    format="{asctime} - {levelname} - {message}",
+    style="{",
+    datefmt="%Y-%m-%d %H:%M",
+)
+
+donuts = 5
+guests = 0
+try:
+    donuts_per_guest = donuts / guests
+except ZeroDivisionError as _:
```

Comment:
We can probably just remove the `as` clause:

```suggestion
except ZeroDivisionError:
```